logstash-core/lib/logstash/api/commands/node.rb

```diff
@@ -62,7 +62,7 @@ def initialize(cmd, options)
           end
           
           def to_s
-            hash = to_hash
+            hash = to_hash[:hot_threads]
```

Comment:
@andrewvc what you saw here is that the hot threads should return a class that actually have a `.to_s` method, the one used to serialize the data as "human", if you return a hash for sure the `to_s` method is not there to convert it to human. same thing or any other changes, at less the way the `respond_with` method is implemented right now.